mkdocs.yml

```diff
@@ -13,7 +13,7 @@ validation:
   not_found: warn
   absolute_links: ignore
 # fails to build on warnings
-strict: true
+# strict: true
```

Comment:
Why are we removing this check? We should fix the issue it finds not remove the check

Reply:
Had to remove it so i could build it locally to test run it. 
Happens when you create a new file, not on existing files. 
But it shouldn't be removed for the live version.
That's why @zakkarry added the revert.